public/scripts/editor/js/bramble-editor.js

```diff
@@ -58,21 +64,8 @@ define(function(require) {
       console.error("[Bramble Error]", err);
     });
 
-    function mount(err, config) {
-      if(err) {
-        console.error("[Bramble Error]", err);
-        return;
-      }
-
-      // Now that fs is setup, tell Bramble which root dir to mount
-      // and which file within that root to open on startup.
-      Bramble.mount(config.root, config.open);
-    }
-
     // Update the Project Title in the UI and allow it to be renamed
+    // TODO: should this be in BrambleUIBradge or Project or a combo?  Seems wrong here.
```

Comment:
The problem here is timing. If we do it in `BrambleUIBridge`, which only initializes after populating the fs, we are blocking on a long operation and will see an empty project title at the top until then. Maybe we should put this in `Project`.